polkadot/node/network/collator-protocol/src/validator_side/tests/mod.rs

```diff
@@ -205,6 +205,7 @@ struct TestHarness {
 
 fn test_harness<T: Future<Output = VirtualOverseer>>(
 	reputation: ReputationAggregator,
+	ah_invulnerable_collators: HashSet<PeerId>,
```

Comment:
I like using `ah_invulnerable_collators: impl Into<HashSet<u32>>` and then you can pass `[]` instead of (longer) `HashSet::new()`, but maybe that is more costly for the compiler?